src/Core/Domain/Entity/TelemetryEvent/TelemetryEventMetadata.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Core\Domain\Entity\TelemetryEvent;
+
+use SprykerSdk\SdkContracts\Entity\Telemetry\TelemetryEventMetadataInterface;
+
+class TelemetryEventMetadata implements TelemetryEventMetadataInterface
+{
+    /**
+     * @var string|null
+     */
+    protected ?string $developerEmail;
+
+    /**
+     * @var string|null
+     */
+    protected ?string $developerGithubAccount;
+
+    /**
+     * @var string|null
+     */
+    protected ?string $projectName;
+
+    /**
+     * @param string|null $developerEmail
+     * @param string|null $developerGithubAccount
+     * @param string|null $projectName
+     */
+    public function __construct(?string $developerEmail, ?string $developerGithubAccount, ?string $projectName)
+    {
+        $this->developerEmail = $developerEmail;
+        $this->developerGithubAccount = $developerGithubAccount;
+        $this->projectName = $projectName;
+    }
+
+    /**
+     * @return string|null
+     */
+    public function getDeveloperEmail(): ?string
```

Comment:
you don't need to enable logic if this data is empty. it doesn't make sense to have the object if metadata is null.